CONTRIBUTING.rst

```diff
@@ -609,4 +609,3 @@ release tag by appending ``^!`` to the commit or tag name. For example to run
 the skimage.filter module benchmarks on release v0.17.2::
 
     asv run -b Filter v0.17.2^!
```

Comment:
```suggestion
    asv run -b Filter v0.17.2^!

```

Reply:
I left the whitespace when editing online because it's currently like this in `main` (I checked locally in vim). I only wanted to bring this PR on par with `main`.